airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -1803,14 +1801,11 @@ public Stream<? extends Arguments> provideArguments(final ExtensionContext conte
           Jsons.deserialize(MoreResources.readResource(NAMESPACE_TEST_CASES_JSON));
       return MoreIterators.toList(testCases.elements()).stream()
           .filter(testCase -> testCase.get("enabled").asBoolean())
-          .map(testCase -> {
-            final String randomSuffix = Strings.addRandomSuffix("", "", 5);
-            return Arguments.of(
-                testCase.get("id").asText(),
-                // Randomise namespace to avoid collisions between tests.
-                testCase.get("namespace").asText() + randomSuffix,
-                testCase.get("normalized").asText() + randomSuffix);
-          });
+          .map(testCase -> Arguments.of(
+              testCase.get("id").asText(),
+              // Add uniqueness to namespace to avoid collisions between tests.
+              TestingNamespaces.generate(testCase.get("namespace").asText()),
+              TestingNamespaces.generate(testCase.get("normalized").asText())));
```

Comment:
Should these be added to `TEST_SCHEMAS` as well so DBs can clean this up at the end of the suite?  Might have missed this one earlier

Reply:
the test where these gets used handles adding them. so there's no need to add them here too!